src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
True - but unless I'm missing something, I'd have to publish the original event type because the callback needs to pass it along. It's a core part of the feature - if you subscribeAll you need to know what the event was.